client/ayon_harmony/plugins/load/load_palette.py

```diff
@@ -8,66 +8,98 @@
 import ayon_harmony.api as harmony
 
 
-class ImportPaletteLoader(load.LoaderPlugin):
-    """Import palettes."""
+class LinkPalette(load.LoaderPlugin):
```

Comment:
Looking at most loader plug-ins across our code base, it's pretty common to end loader plugin class names with `Loader`, so I'd stick to that convention here too.
```suggestion
class LinkPaletteLoader(load.LoaderPlugin):
```